projects/packages/search/changelog/update-search-record-meter-remove-feature-flag

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: removed
+
+Record Meter: removes feature flag functionality so the record meter is available to all users
```

Comment:
I reckon we have to think about what would be most useful in the release changelog here. From the user's point on view, we've added a new feature. So perhaps type = added, and description as suggested?
```suggestion
Record Meter: make feature available to all users
```